lib/cell.rb

```diff
@@ -32,33 +32,33 @@ def fire_upon
 
   end
 
-def render_fire_upon_helper
-  case ship
-  when nil
-    p "M"
-  else
-    if ship.sunk?
-      p "X"
+  def render_fire_upon_helper
+    case ship
+    when nil
+      "M"
     else
-      p "H"
+      if ship.sunk?
+        "X"
+      else
+        "H"
+      end
     end
   end
-end
 
   def render(show_ships = false)
     case @fired_upon
     when false
       if ship == nil
-        p "."
+        "."
       else
         if show_ships == true
-          p "S"
+          "S"
         else
-          p "."
+          "."
```

Comment:
Removed all the "p"s as it made our board print the render above the actual board in the terminal